resources/gameResources/snapchat-filters.json

```diff
@@ -0,0 +1,39 @@
+{
```

Comment:
ok, so i'm guessing these are the synonyms. they should live in the code and parsing it from file is even less nice than from code. if possible have them as final static variables inside corresponding states cause they're very use-case specific

Reply:
the roboy one is pretty generic, but the rest not so much